CHANGES.md

```diff
@@ -74,6 +74,7 @@
 
 * If a main session fails to load, the pipeline will now fail at worker startup. ([#25401](https://github.com/apache/beam/issues/25401)).
 * Python pipeline options will now ignore unparsed command line flags prefixed with a single dash. ([#25943](https://github.com/apache/beam/issues/25943)).
+* Support of Cloud Debugger service ([deprecated](https://cloud.google.com/debugger/docs/deprecations)) is removed from Dataflow runner. CloudDebugOptions has no effect now and is subject to remove in nearly future. (Java) ([#25959](https://github.com/apache/beam/issues/25959)).
```

Comment:
2.47.0? we might need to wait for 2.48.0 or even 2.49.0

Reply:
in this case removed milestone and adjusted priority.

Note that the service is scheduled to shutdown on May 31 which is between the release of 2.47.0 (end Apr) and 2.48.0 (mid June), which means it would be removed in no later than v2.48.0.

Reply:
My vote would be to release a couple versions where the option explicitly throws a nice error explaining why it no longer works. Regardless, I agree that specifying the specific release it will be removed in would be nice - I'd vote we say 2.48.0

Reply:
2.47.0: add warning message about removing this from 2.48.0 
2.48.0: remove it
Is this ok?

Reply:
SGTM